config/routes.rb

```diff
@@ -37,6 +37,9 @@
         post 'add_meal', to: 'branches#create_meal'
         get :toggle_meal_active
         get :toggle_meal_inactive
+        post :order_ready
+        post :order_picked_up
+        post :order_cancelled
```

Comment:
i think we should add these routes on orders as we already have branch id there.
Just authorize admin before these actions.

Reply:
Done